src/esm_tests/repos.py

```diff
@@ -30,6 +35,33 @@ def update_resources_submodule(info, verbose=True):
             logger.error(f"Pull of {resources_branch} branch not possible")
             sys.exit(1)
 
+def check_resources(info, verbose=True):
+    resources_folder = f"{info['script_dir']}/resources/"
+    dir_resources = os.listdir(resources_folder)
+    if not (
+        "last_tested" in dir_resources
+        and "runscripts" in dir_resources
+        and not info.get("repo_update", False)
+    ):
+        install_resources = questionary.select(
```

Comment:
I think `questionary` has a True/False selector, so you might not need to look below for the explicit string `"Yes"`. That's small stuff though, if it works, leave it alone :-)

Reply:
solved :)